src/CMakeLists.txt

```diff
@@ -141,6 +141,7 @@ SET(LMMS_REQUIRED_LIBS ${LMMS_REQUIRED_LIBS}
 	${SAMPLERATE_LIBRARIES}
 	${SNDFILE_LIBRARIES}
 	${EXTRA_LIBRARIES}
+	${FFTW3F_LIBRARIES}
```

Comment:
Really minor, but I think it'd be better if this line is before `${EXTRA_LIBRARIES}`.

Reply:
Done.